### DISCUSSION TITLE
Renovacion visa TN en Tijuana

### DISCUSSION THREAD

**Discussion by pedroxido** [pollero/pollero#63] [Q&A]
Hola comunidad! 
Actualmente vivo en Los Angeles y estoy planeando renovar mi visa TN en los prox 4 meses. Sinceramente no quisiera ir hasta CDMX y preferiria conducir desde Los Angeles hasta Tijuana pero no logro encontrar informacion con respecto a lo siguiente:

1. Estan procesando/emitiendo visas TN en Tijuana
2. Los tiempos de espera para agendar una cita : https://travel.state.gov/content/travel/en/us-visas/visa-information-resources/wait-times.html << este sitio por experiencia no suele ser siempre certero, solo sirve como estimacion

Usualmente lo que yo solia hacer es llenar la DS160, luego completar el formulario + pago PERO pues debe haber otra manera de saber los tiempos de espera sin tener que llenar multiples DS-160, verdad?
Cualquier comentario ayuda es bienvenida!

**Comment by SergioEduardoAR** [comment]
Por ahora los únicos consulados donde procesan TN son Ciudad de México, Guadalajara y Ciudad Juárez.
